### ISSUE TITLE
Add option to clear co-authors after commit

### ISSUE THREAD

**Issue by tobiasdiez** [rkotze/git-mob-vs-code#43]
## Summary

After a commit, the co-authors are automatically added to the next commit. This makes sense in the context of a mob session. However, for more sporadic collaboration where you have only a few commits that are shared with others its more convenient to remove the coauthors after a commit.

## Motivation
see above

## Additional context
Given the scope of this extension, I wouldn't change the default behavior and only add an option that toggles the clearance of authors after a commit.

**Comment by rkotze** [comment]
Thanks for the suggestion. I will have a look into building this feature.

**Comment by rkotze** [comment]
👋  @tobiasdiez 

I've had a look into this and so far the best thing I've come up with is to use the `post-commit` git hook.

```javascript
let exec = require('child_process').exec;

exec("git solo");
```

However, this does not play well with updating the `git-mob` UI. You can give it a try.

I welcome any suggestions, I do think it would be neat to have this feature.

**Comment by tobiasdiez** [comment]
Thanks for looking into this. 

I have not much experience with vs code extensions, but what about listening to the `onDidChange` events of the git extension?
https://github.com/microsoft/vscode/blob/master/extensions/git/src/api/git.d.ts#L99-L116.

There is also `onDidRunOperation` which is invoked directly after a commit, but it seems its not exposed in the API. https://github.com/microsoft/vscode/blob/master/extensions/git/src/repository.ts#L572

**Comment by rkotze** [comment]
Thanks for looking. 

I've tried `onDidChange` for repositories and that is triggered for a range of actions, like staging and committing. However, it's impossible to tell which caused it to trigger as there is no operation data passed it.

For `onDidRunOperation` is not available from the git ext API, but that would have been good option.

**Comment by rkotze** [comment]
I've created a stackoverflow question https://stackoverflow.com/questions/63306762/update-extension-ui-after-a-git-commit-in-vs-code

**Comment by rkotze** [comment]
@tobiasdiez i've released version 1.7 to address this. Give it a try when you can and let me know how it works for you. Thanks.
